docs/guides/installation-ami.md

```diff
@@ -2,6 +2,9 @@
 
 This document walks you through deploying the TriggerMesh AMI (beta) on the Amazon AWS platform.
 
+!!! warning
```

Comment:
> :warning: **Warning**: This installation method is meant to provide a fast mechanism to test TriggerMesh on AWS. It is not meant to be used in a production environment.

@sebgoa should we use this formatting for warnings?

Reply:
@sameersbn this is rendered by a plugin, that's why it doesn't show was valid Markdown here.